nixos/modules/programs/_1password-gui.nix

```diff
@@ -8,19 +8,16 @@ let
 
 in
 {
+  imports = [
+    (mkRemovedOptionModule [ "programs" "_1password-gui" "gid" ] ''
+      A preallocated GID would be used instead.
```

Comment:
Why "would" and not "will"? Is this hypothetical or conditional in any way?

Reply:
I changed it to "will." It only depends the module being enabled.